src/pages/Login/Login.scss

```
@@ -0,0 +1,58 @@
+.login {
+  .login_wrap {
+    width: 516px;
+    height: 407px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  input {
+    width: 380px;
+    height: 60px;
+    padding: 11px 13px !important;
+    margin-bottom: 16px !important;
+    background-color: #fafafa;
+    border-color: #dddddd;
+    border-style: solid;
+    border-radius: 3px;
+    border-width: 1px;
+  }
+
+  button {
+    width: 375px;
+    height: 60px;
+    padding: 11px 13px !important;
+    margin-bottom: 16px !important;
+    background-color: #fafafa;
+    border-color: #09388c;
+    border-style: solid;
+    border-radius: 3px;
+    border-width: 1.5px;
```

border-color, border-style, border-width는 한줄에 선언 가능합니다!
```suggestion
    border: 1.5px solid #09388c
    border-radius: 3px;
```